### PR TITLE
Improve image testing

### PR DESCRIPTION
**Pull request by loliee**
This PR brings some testing enhancements, commits should speak by themself. 
Hopefully it's 🇨🇭🇩🇪

### CODE REVIEW

**Comment by fauust** [comment]
Regarding 39f3feb and after reading https://linuxhint.com/debian_frontend_noninteractive/ and https://github.com/MariaDB/mariadb.org-tools/issues/43, don't you think that this would be better :

```bash
RUN DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y package
```

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,18 @@
 FROM debian:10
 LABEL maintainer="faustin@fala.red"
 
-ENV DEBIAN_FRONTEND noninteractive
-
-RUN apt-get update \
-    && apt-get install --no-install-recommends -y systemd systemd-sysv \
-    python3 sudo bash ca-certificates python3-apt iproute2 xz-utils \
+RUN set -eux \
+    apt-get update \
```
Comment (fauust) [inline Dockerfile:5]:
```suggestion
    && apt-get update \
```

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,19 @@
 ---
+language: minimal
+
 services: docker
 
 before_install:
-  - sudo apt-get update
   - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+  - wget https://github.com/wagoodman/dive/releases/download/v0.9.2/dive_0.9.2_linux_amd64.deb
+  - sudo apt install ./dive_0.9.2_linux_amd64.deb
```
Comment (fauust) [inline .travis.yml:9]:
```suggestion
  - sudo apt-get install ./dive_0.9.2_linux_amd64.deb
```

File: Dockerfile (hunk 1 above)
Comment (loliee) [reply Dockerfile:5]: 😅

**Pull request merged**